### PR TITLE
first pass operations doc

### PR DESCRIPTION
**Pull request by christian-posta**
TODO:

Fill out Production Lockdown doc a bit more

Add debugging the control plane to the debug doc

cc @lgadban @bdecoste

### CODE REVIEW

**Comment by christian-posta** [comment]
This PR is WIP... sent in for visibility

File: docs/content/operations/debugging_gloo/_index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Debugging Gloo
+description: This document shows how some common ways to debug Gloo and Envoy
+weight: 10
+---
+
+At times you may need to debug Gloo and misconfigurations. Gloo is based on Envoy and often times these misconfigurations are observed as a result of behavior seen at the proxy. This guide will help you debug issues with Gloo and Envoy. 
+
+
```
Comment (lgadban) [inline docs/content/operations/debugging_gloo/_index.md]: Might be worth mentioning `glooctl check` as an entry point to diagnosing problems

Comment (christian-posta) [reply docs/content/operations/debugging_gloo/_index.md]: Yes! Will do right now, thanks

File: docs/content/operations/debugging_gloo/_index.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
+---
+title: Debugging Gloo
+description: This document shows how some common ways to debug Gloo and Envoy
+weight: 10
+---
+
+At times you may need to debug Gloo and misconfigurations. Gloo is based on [Envoy](https://www.envoyproxy.io) and often times these misconfigurations are observed as a result of behavior seen at the proxy. This guide will help you debug issues with Gloo and Envoy.
```
Comment (ashleywang1) [inline docs/content/operations/debugging_gloo/_index.md]:
```suggestion
At times you may need to debug Gloo misconfigurations. Gloo is based on [Envoy](https://www.envoyproxy.io) and often times these misconfigurations are observed as a result of behavior seen at the proxy. This guide will help you debug issues with Gloo and Envoy. 
```

File: docs/content/operations/debugging_gloo/_index.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+### Changing logging Levels and more
+
+Each Gloo control plane component comes with a [optional debug port]({{< versioned_link_path fromRoot="/observability/ports/" >}}) that can be enabled with the `START_STATS_SERVER` environment variable. To get access to it, you can port-forward to it with Kubernetes like this:
+
+```bash
+kubectl port-forward  -n gloo-system deploy/gloo  9091:9091
+```
+
+Now you can navigate to http://localhost:9091 and you get a simple page with some additional endpoints:
```
Comment (ashleywang1) [inline docs/content/operations/debugging_gloo/_index.md]:
```suggestion
Now you can navigate to [http://localhost:9091](http://localhost:9091) and you get a simple page with some additional endpoints:
```

Comment (christian-posta) [reply docs/content/operations/debugging_gloo/_index.md]: I'll commit the suggestion, but doesn't MD automatically turn that into URLs now?

File: docs/content/operations/production_deployment/_index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+This document shows some of the Production options that may be useful. We will continue to add to this document and welcome users of Gloo to send PRs to this as well.
+
+
+## Dropping capabilities
+
+One of the more important (and unique things about Gloo) is the ability to significantly lock down the edge proxy. Other proxies require privileges to write to disk or access the Kubernetes API, while Gloo splits those responsibilities between control plane and data plane. The data plane can be locked down with zero privileges while separating out the need to secure the control plane differently. 
+
+For example, Gloo's data plane (the `gateway-proxy` pod) has ReadOnly file system. Additionally it doesn't require any additional tokens mounted in or OS-level privileges. By default some of these options are enabled to simplify developer experience, but if your usecase doesn't need them, you should lock them down.
```
Comment (ashleywang1) [inline docs/content/operations/production_deployment/_index.md]:
```suggestion
For example, Gloo's data plane (the `gateway-proxy` pod) has ReadOnly file system. Additionally it doesn't require any additional tokens mounted in or OS-level privileges. By default some of these options are enabled to simplify developer experience, but if your use case doesn't need them, you should lock them down. 
```

File: docs/content/operations/debugging_gloo/_index.md (hunk 3 above)
Comment (ashleywang1) [reply docs/content/operations/debugging_gloo/_index.md]: Oh, it does! I'll remove it.

**Pull request merged**